README.md

```diff
@@ -57,6 +59,9 @@ To deploy workloads with SR-IOV VF or PCI PF, this plugin needs to work together
   - Host device CNI (for PCI PFs)
     - During Pod creation, plumbs the allocated network device to the Pods network namespace using device information given by the meta plugin
     - On Pod deletion, reset and release the allocated network device from the Pod
+  - Ovn-kubernetes (for SFs Auxiliary netdevices and Switchdev VFs)
```

Comment:
This work is already present upstream ?

Reply:
No :see_no_evil:
But we always can merge this PR after, right?

Reply:
it can be a follwup PR to update doc once PR is merged in ovn-kubernetes :)

Reply:
can you remove this section ? once OVN-Kubernetes PR lands this section can be updated